src/test/java/org/testobject/appium/junit/RemoteWebDriverCalculatorSuiteTestJUnit.java

```diff
@@ -33,7 +33,7 @@ public void setup() throws MalformedURLException {
         capabilities.setCapability(TestObjectCapabilities.TESTOBJECT_API_KEY, watcher.getApiKey());
         capabilities.setCapability(TestObjectCapabilities.TESTOBJECT_TEST_REPORT_ID, watcher.getTestReportId());
 
-        driver = new RemoteWebDriver(TestObjectCapabilities.TESTOBJECT_APPIUM_ENDPOINT, capabilities);
+        driver = new RemoteWebDriver(watcher.getTestObjectOrLocalAppiumEndpointURL(), capabilities);
```

Comment:
Changing the method name will require users to change their test code. I agree it's ugly, though.

I think adding javadoc for the method or changing it's name should be in a separate PR, since it's not really related to these changes.